faceted_search/src/extension/popup.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+class PopUp extends React.Component{
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className="popBox">
+        <ui>
+          <li>Using "<b>*</b>" for the wildcard search (eg. IGSN*).</li>
+          <li><b>&&</b> for the AND operator, <b>||</b> for the OR operator.</li>
+          <li>Using parenthesis "<b>()</b>" to group the multiple keywrods (eg. (water && great)).</li>
+          <li>If there are special characters (eg. : ) in the search words, Please put entire string in a quote, (eg. "IGSN:NHB0005J0").</li>
```

Comment:
We could change the hint information if we need to tell users more or fewer details.